assets.css

```diff
@@ -2,7 +2,33 @@
  * Restrict sizing to the page width in all browsers (opinionated).
  */
 
-:where(iframe, img, input, video, select, textarea) {
+:where(iframe, input, video, select, textarea) {
   height: auto;
   max-width: 100%;
 }
+
+/**
+ * Do not stretch image or grow bigger than parent
+ */
+
+:where(img) {
+  max-width: 100%;
+}
+
+/**
+ * Preserve aspect ratio
+ */
+
+:where(img[width][height]) {
+  height: auto;
+}
+
+/**
+ * Let SVG scale without boundaries
+ */
+
+:where(img[src$=".svg"]) {
+  width: 100%;
+  height: auto;
+  max-width: none;
+}
```

Comment:
Could we move this bit into a discussion? I’m unsure about it, mainly because .svg seems like such a fragile way to identify the image.

Reply:
I think if we let this piece out it can cause some strange side affects with SVG images. In the article it doesn't say anything about these lines of code but I think it is part of the whole image solution. Looks like they did put some thought into it.

Reply:
`Width: 100%` seems pretty opinionated to me. SVGs _are_ scalable, but they also define their inherent default dimensions the user would then have to revert the rule to.

Reply:
You are right @kernc. I also did various tests and I think it is pretty save to leave the SVG part out. I will change the PR.